cmd/root.go

```diff
@@ -95,6 +99,7 @@ their cloud accounts.`,
 	flags.StringVarP(&rO.config, "config", "c", "", "Config file (default is https://github.com/run-x/cloudgrep/blob/main/pkg/config/config.yaml)")
 	flags.StringVar(&rO.bind, "bind", "", "Host to bind on")
 	flags.StringVarP(&rO.regions, "regions", "r", "", "Comma separated list of regions to scan, or \"all\"")
+	flags.StringVarP(&rO.profiles, "profiles", "", "", "Comma separated list of AWS profiles to scan.")
```

Comment:
suggestion(non-blocking): We can use `flags.StringSliceP` to split the values automatically, and to support multiple `--profiles` options (e.g. `--profiles=one,two --profiles=three`). Ditto for `--regions`, but that can be left alone as it's not part of this PR.

Reply:
this is good, I forgot about this